### ISSUE TITLE
civic.url is ignored

### ISSUE THREAD

**Issue by ruslan-forostianov** [cBioPortal/cbioportal#9503]
There is `civic.url` property to set CIVIC url.
It's documented here.
https://github.com/cBioPortal/cbioportal/blob/master/docs/portal.properties-Reference.md#civic-integration

It seems to be ignored by the frontend atm.

Check https://github.com/cBioPortal/cbioportal-frontend/blob/master/packages/cbioportal-utils/src/civic/CivicDataFetcher.ts

**Comment by alisman** [comment]
@ruslan-forostianov we are wondering, are you actually planning to make use of this? i.e. point at another instance of civic?  i ask because fixing this will require some annoying refactoring to feed the url.   my guess, since you are asking about it, is that you ARE going to use it!  But just wanted to understand the use case.

**Comment by ruslan-forostianov** [comment]
@alisman Yes, we are using local civic instance to prevent a potential leakage of sensitive data.

We achieve that by replacing the civic public url with our custom url in the js code with sed on cbioportal start.

**Comment by alisman** [comment]
@alisman ok, we will try to fix this.
